packages/typespec-ts/test/util/emitUtil.ts

```diff
@@ -207,9 +210,13 @@ export async function emitParameterFromTypeSpec(
     libraryName: "test",
     schemas: [],
     parameters,
+    helperDetails,
     importInfo: {
       internalImports: importSet,
       runtimeImports: buildRuntimeImports("azure")
+    },
+    options: {
+      sourceFrom: "TypeSpec"
```

Comment:
is the default not TypeSpec?

Reply:
if we didn't set this this would be `undefined`. i just set it explicitly in code.

in our code we would set it as TypeSpec.